src/Menu/menu.scss

```
@@ -1,45 +1,45 @@
 .menu {
   position: relative;
 }
+
 .menu-content {
   position: absolute;
   top: 100%;
   z-index: 10;
-  background: #fff;
+  background: var(--pgn-color-white, $white);
```

This *looked like* a breaking change to me at first, but since `#white` is being redefined as `#fff` in `index.scss`, and this file is included *via* `index.scss`, that should override any brand overrides of `$white`. 

That being said, it's also possible someone has imported this `menu.scss` file somewhere and doesn't have `$white` defined, so I think falling back to the hardcoded value for now may be the safest option

```suggestion
  background: var(--pgn-color-white, #fff);
```